homework1/task04/tests/test_sum_of_four.py

```diff
@@ -0,0 +1,51 @@
+"""
+This is test_sum_of_four.py docstring.
+
+Here are tests for check_sum_of_four function
+"""
+
+
+import pytest
+from sum_of_four.task04 import check_sum_of_four
+
+
+@pytest.mark.parametrize(
+    ("a", "b", "c", "d", "expected_result"),
+    [
+        (
+            [1, -2, 3, 4, -5],
```

Comment:
I'd recommend using small and easy-to-understand examples.   The rule of thumb: use a test case that can be calculated in your head.

Reply:
Yes, I agree.  Sometimes I am enthysiastic to check something big :D

Reply:
Please change it to something smaller.

Each case should represent something new.  Normal value, all empty, all zeros. 
In general, you don't need to have multiple cases for normal

Reply:
Fixed